### ISSUE TITLE
Missing label for the page selection dropdown

### ISSUE THREAD

**Issue by tsv2013** [surveyjs/survey-creator#857]
See the https://surveyjs.answerdesk.io/ticket/details/t4494/accessibility-issue-for-pagination-dropdown-ensure-every-form-element-has-a-label thread for more details
